draft-smyslov-ipsecme-rfc8229bis.xml

```diff
@@ -1102,15 +1102,11 @@
   2)  ---------------------  TLS Session  ---------------------
       ClientHello               ---------->
                                                     ServerHello
-                                                   Certificate*
-                                             ServerKeyExchange*
-                                <----------     ServerHelloDone
-      ClientKeyExchange
-      CertificateVerify*
-      [ChangeCipherSpec]
-      Finished                  ---------->
-                                             [ChangeCipherSpec]
-                                <----------            Finished
+                                          {EncryptedExtensions}
+                                                 {Certificate*}
```

Comment:
Same with server's reply - shouldn't it include either key_share or preshared_key? What about CertificateRequest?